### ISSUE TITLE
Email: ITI Comments on First Public Working Draft of W3C Accessibility Guidelines (WCAG) 3.0 (10 - Usability improvements to TR)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#452]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0012.html): 
from [PDF](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/att-0012/ITI_Comments_on_WCAG_3.0_First_Public_Working_Draft_20210224_1_.pdf)

2.	There is considerable introductory information, including information about the guidelines structure, testing, and scoring. Are there usability improvements that would make it easier to use and find information?

Yes - the guidelines structure should be in an explainer document and this document focus on the requirements. Testing and scoring should provide better practical usage with examples, as it is hard to comprehend the bigger picture that leads to the overall conformance and score.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
